spec/lookup_spec.rb

```diff
@@ -17,8 +15,8 @@
     describe "#initialize(table_name)" do
       it "calls create on lookup_table" do
         table = double()
-        table.should_receive(:table_name)
-        subject.new(table)
+        expect(table).to receive(:table_name)
+        subject.new(table, nil)
```

Comment:
@samfrench I'd also probably just call `subject.new(table)` and not bother with passing `nil` as the second argument.